examples/plot-graph-using-bokeh.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Is the name of the notebook reviewed by the PM?

Reply:
Yes.